source/as-promise/types.ts

```diff
@@ -126,10 +126,11 @@ export interface PaginationOptions<ElementType, BodyType> {
 		requestLimit?: number;
 
 		/**
-		Defines how the parameter `allItems` in pagination.paginate, pagination.filter and pagination.shouldContinue is managed.
-		When set to `false`, the parameter `allItems` is always an empty array.
+		Defines how the property `allItems` in pagination.paginate, pagination.filter and pagination.shouldContinue is managed.
+
+		By default, the property `allItems` is always an empty array. This setting can be helpful to save on memory usage when working with a large dataset.
```

Comment:
@szmarczak Don't forget to keep the TS docs in sync.

Reply:
Ah sorry, overlooked this one

Reply:
Fixed: https://github.com/sindresorhus/got/commit/4cce4de19ed577f481ba45a023b395a29cadce86

Reply:
Thanks, currently I'm not home. I'll be back tomorrow.